### PR TITLE
[dagit] add log-level flag

### PR DESCRIPTION
**Pull request by alangenfeld**
### Summary & Motivation

make the hard coded uvicorn args configurable via cli

### How I Tested These Changes

```
dagit --log-level info
2022-05-12 09:15:21 -0500 - dagit - INFO - Serving dagit on http://127.0.0.1:3000 in process 27231
INFO:     Started server process [27231]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://127.0.0.1:3000 (Press CTRL+C to quit)
INFO:     127.0.0.1:49316 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49316 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49316 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49317 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49318 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49317 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49318 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49319 - "POST /graphql HTTP/1.1" 200 OK
INFO:     ('127.0.0.1', 49320) - "WebSocket /graphql" [accepted]
INFO:     ('127.0.0.1', 49322) - "WebSocket /graphql" [accepted]
INFO:     ('127.0.0.1', 49323) - "WebSocket /graphql" [accepted]
INFO:     127.0.0.1:49324 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49324 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49324 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49324 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49325 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49327 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49328 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49329 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49325 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49329 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49326 - "POST /graphql HTTP/1.1" 200 OK
INFO:     127.0.0.1:49327 - "POST /graphql HTTP/1.1" 200 OK
^C
Aborted!
INFO:     Shutting down
INFO:     Waiting for application shutdown.
INFO:     Application shutdown complete.
INFO:     Finished server process [27231]
```
`dagit --log-level INFO`
`dagit --log-level InFo`

### CODE REVIEW

**Comment by alangenfeld** [comment]
Current dependencies on/for this PR:
* master
  * **PR #7853** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/7853" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/7853?utm_source=stack-comment).

File: python_modules/dagit/dagit/cli.py [hunk 1]
```diff
@@ -97,8 +97,24 @@ def create_dagit_cli():
     help="Filter all warnings when hosting Dagit.",
     is_flag=True,
 )
+@click.option(
+    "--log-level",
+    help="Set the log level.",
+    show_default=True,
+    default="warning",
+    type=click.Choice(["critical", "error", "warning", "info", "debug", "trace"]),
```
Comment (gibsondan) [inline python_modules/dagit/dagit/cli.py:105]: sometimes these are CAPS no?

**Review by gibsondan — APPROVED** [review]
sweet - could be nice to have this on the daemon too

File: python_modules/dagit/dagit/cli.py (hunk 1 above)
Comment (alangenfeld) [reply python_modules/dagit/dagit/cli.py]: good call updated

**Pull request merged**